tox.ini

```diff
@@ -24,4 +30,5 @@ commands =
     pytest --cov dsctriage dsctriage
 
 [flake8]
-max-line-length = 120
\ No newline at end of file
+max-line-length = 120
+extend-ignore = E203
```

Comment:
A little annoying that flake8 and black disagree on whitespace before ':' when it comes to list slicing, but this should be fine